Plotting with VegaLite does not work on Windows 11

## Environment

<!-- Note: you can also find versions at the /dashboard route of your Livebook. -->

* Elixir & Erlang/OTP versions (elixir --version):
    ```
    Erlang/OTP 24 [erts-12.1.4] [source] [64-bit] [smp:16:16] [ds:16:16:10] [async-threads:1] [jit]

    Elixir 1.13.3 (compiled with Erlang/OTP 22)
    ```
* Operating system: Windows 11
* How have you started Livebook (mix phx.server, livebook CLI, Docker, etc): `livebook server --port 8086` through Windows Terminal (PowerShell)
* Livebook version (use `git rev-parse HEAD` if running with mix): `v0.5.2`
* Browsers that reproduce this bug (the more the merrier): Edge
* Include what is logged in the browser console:
    ![image](https://user-images.githubusercontent.com/65685447/154867966-a8747fc4-54cf-45d5-972e-342bf3ee90cf.png)

* Include what is logged to the server console: Nothing.

## Current behavior

I currently cannot get plotting to work with VegaLite when running Livebook in Windows. There's no plot and no error is reported in the Livebook notebook. This error reproduces with the "Plotting with VegaLite" notebook included with Livebook in the Explore section.

![image](https://user-images.githubusercontent.com/65685447/154868064-6fd623d9-1183-4b3b-9296-32a97df08d11.png)

## Expected behavior

A plot should be shown. I don't think this is a browser issue because I downloaded a Docker image as instructed on `livebook.dev`, and when running Livebook from Docker in Windows and then navigating to the application in Edge still in Windows, the plotting works as expected.

![image](https://user-images.githubusercontent.com/65685447/154868126-48f1a0f4-5781-4419-99e0-bed3409b3a7b.png)

I was running the `mix escript.install github livebook-dev/livebook` command.

With that method,
```powershell
> Invoke-WebRequest http://localhost:8081/iframe/v2.html -Method Get
Invoke-WebRequest :

No Numbats here!
At line:1 char:1
+ Invoke-WebRequest http://localhost:8081/iframe/v2.html -Method Get
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [In
   voke-WebRequest], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.Invo
   keWebRequestCommand
```
That matches up with what the browser tools report.
![image](https://user-images.githubusercontent.com/65685447/155035771-9a6b7beb-2db6-4c04-a4c6-4f251cbea6a0.png)

---------------------------

I went ahead and cloned the `livebook` repo with `git` and ran things locally. Using that method, the following error is constantly reported to the console:

```powershell
Error: Cannot find module 'c:\Users\<user>\Desktop\Livebook Test\livebook\assets\node_modules\webpack\bin\webpack.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
[error] Task #PID<0.799.0> started from LivebookWeb.Endpoint terminating
** (stop) :watcher_command_error
    (phoenix 1.6.6) lib/phoenix/endpoint/watcher.ex:55: Phoenix.Endpoint.Watcher.watch/2
    (elixir 1.13.3) lib/task/supervised.ex:89: Task.Supervised.invoke_mfa/2
    (stdlib 3.16.1) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Function: &Phoenix.Endpoint.Watcher.watch/2
    Args: ["node", ["node_modules/webpack/bin/webpack.js", "--mode", "development", "--watch", "--watch-options-stdin", {:cd, "c:/Users/<user>/Desktop/Livebook Test/livebook/assets"}]]
node:internal/modules/cjs/loader:936
  throw err;
  ^
```
With the Livebook server running that way, running the same cell that hasn't generated errors inside the notebook now gives an error:
![image](https://user-images.githubusercontent.com/65685447/155035526-dde58c73-5cdc-42c6-bf22-e2b39c9be4e1.png)

Lastly, running `File.read!("iframe/priv/static/iframe/v2.html")` in `iex` running at the root of the Livebook project returns a string with only newlines, either just `\n` or pairs `\n\n`. A snippet:

```powershell
\livebook> iex.bat
Interactive Elixir (1.13.3) - press Ctrl+C to exit (type h() ENTER for help)
iex(1)> File.read!("iframe/priv/static/iframe/v2.html")
"<!DOCTYPE html>\n<html lang=\"en\">\n  <head>\n    <title>Output</title>\n    <style>\n      html,\n      body {\n        margin: 0;\n        padding: 0;\n        font-family: sans-serif;\n        overflow-y: hidden;\n      }\n    </style>\n  </head>\n  <body>\n    <div id=\"root\"></div>\n    <script>\n      \"use strict\";\n\n      // Invoke the init function in a separate context for better isolation\n      function applyInit(init, ctx, data) {\n        init(ctx, data);\n      }\n\n      (() => {\n        const state = {\n          token: null,\n          importPromise: null,\n          eventHandlers: {},\n
```

--- a/lib/livebook/session.ex
+++ b/lib/livebook/session.ex
@@ -1307,7 +1307,14 @@ defmodule Livebook.Session do
   end
 
   defp extract_archive!(binary, path) do
-    :ok = :erl_tar.extract({:binary, binary}, [:compressed, {:cwd, String.to_charlist(path)}])
+    case :erl_tar.extract({:binary, binary}, [:compressed, {:cwd, String.to_charlist(path)}]) do
+      :ok ->
+        :ok
+
+      {:error, reason} ->
+        File.rm_rf!(path)
+        raise "failed to extract archive to #{path}, reason: #{inspect(reason)}"
+    end
   end
 
   @doc """

